ui/app/helpers/utils/transactions.util.js

```diff
@@ -155,6 +155,8 @@ export function sumHexes(...args) {
   const total = args.reduce((acc, base) => {
     return addCurrencies(acc, base, {
       toNumericBase: 'hex',
+      aBase: 10,
+      bBase: 10,
     })
```

Comment:
This function is called `sumHexes`, and both bases should be `16`. Following other suggestions in this review, the second parameter, `base`, should be renamed to `hexAmount`.